packages/react-radio/src/stories/RadioGroupHorizontal.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { Radio, RadioGroup } from '../index';
+
+export const Horizontal = () => (
+  <RadioGroup layout="horizontal">
```

Comment:
i like the use of horizontal in this control. It makes sense as the parent of items.